classes/invitation/invitations/reviewerAccess/handlers/ReviewerAccessInviteUIController.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+namespace PKP\invitation\invitations\reviewerAccess\handlers;
+
+use APP\core\Application;
+use APP\core\Request;
+use APP\facades\Repo;
+use APP\template\TemplateManager;
+use Nette\Utils\DateTime;
+use PKP\context\Context;
+use PKP\core\Core;
+use PKP\facades\Locale;
+use PKP\invitation\core\enums\InvitationTypes;
+use PKP\invitation\core\Invitation;
+use PKP\invitation\core\InvitationUIActionRedirectController;
+use PKP\invitation\invitations\reviewerAccess\resources\ReviewerAccessInviteResource;
+use PKP\invitation\invitations\reviewerAccess\steps\ReviewerAccessInvitationSteps;
+use PKP\user\User;
+
+class ReviewerAccessInviteUIController extends InvitationUIActionRedirectController
+{
+    protected Invitation $invitation;
+
+    /**
+     * @param Invitation $invitation
+     */
+    public function __construct(Invitation $invitation)
+    {
+        $this->invitation = $invitation;
+    }
+
+    public function createHandle(Request $request, $userId = null): void
+    {
+        if(!$request->getUserVars()['submissionId'] || !$request->getUserVars()['reviewRoundId']) {
+            throw new \Symfony\Component\HttpKernel\Exception\NotFoundHttpException();
+        }
+        $submission = Repo::submission()->get((int) $request->getUserVars()['submissionId']);
+        if(!$submission){
+            throw new \Symfony\Component\HttpKernel\Exception\NotFoundHttpException();
+        }
+
+        $invitationPayload = [
```

Comment:
Why do we have to create this by defining all of the params one by one? Can't we use the invitation's payload instead?